.deepsource.toml

```diff
@@ -3,5 +3,5 @@ version = 1
 [[analyzers]]
 name = "rust"
 
-  [analyzers.meta]
-  msrv = "stable"
\ No newline at end of file
+[analyzers.meta]
```

Comment:
I think Ill just delete deepsource, this is useless and I mostly ignore it. we will see